File: app/src/main/java/com/example/newbabyproject/Visit/ResultVisit.java
package com.example.newbabyproject.Visit;

public class ResultVisit {

    public String result;

    public String userId;
    public String userName;
    public String userPhone;
    public String babyName;
    public String babyNum;
    public String babyRelation;

    private String parentId;
    private String parentName;
    private String visitNotice;
    private String babyWeight;
    private String babyLactation;
    private String babyRequireItem;
    private String babyEtc;
    private String boardConfirm;
    private String writeDate;
    private String replyCnt;
    private String path;
    private String insertDate;
    public ResultVisit(String userId, String userName, String userPhone, String babyName, String babyNum, String babyRelation) {
        this.userId = userId;
        this.userName = userName;
        this.userPhone = userPhone;
        this.babyName = babyName;
        this.babyNum = babyNum;
        this.babyRelation = babyRelation;
    }

    public ResultVisit(String parentId, String parentName, String visitNotice, String babyWeight, String babyLactation, String babyRequireItem, String babyEtc,  String writeDate, String boardConfirm, String path, String replyCnt, String insertDate) {
        this.parentId = parentId;
        this.parentName = parentName;
        this.visitNotice = visitNotice;
        this.babyWeight = babyWeight;
        this.babyLactation = babyLactation;
        this.babyRequireItem = babyRequireItem;
        this.babyEtc = babyEtc;
        this.writeDate = writeDate;
        this.boardConfirm = boardConfirm;
        //this.path = path;
        //this.replyCnt = replyCnt;
        this.insertDate = insertDate;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getUserPhone() {
        return userPhone;
    }

    public void setUserPhone(String userPhone) {
        this.userPhone = userPhone;
    }

    public String getBabyName() {
        return babyName;
    }

    public void setBabyName(String babyName) {
        this.babyName = babyName;
    }

    public String getBabyNum() {
        return babyNum;
    }

    public void setBabyNum(String babyNum) {
        this.babyNum = babyNum;
    }

    public String getBabyRelation() {
        return babyRelation;
    }

    public void setBabyRelation(String babyRelation) {
        this.babyRelation = babyRelation;
    }

    public String getParentId() {
        return parentId;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public String getParentName() {
        return parentName;
    }

    public void setParentName(String parentName) {
        this.parentName = parentName;
    }

    public String getWriteDate() {
        return writeDate;
    }

    public void setWriteDate(String writeDate) {
        this.writeDate = writeDate;
    }

    public String getVisitNotice() {
        return visitNotice;
    }

    public void setVisitNotice(String visitNotice) {
        this.visitNotice = visitNotice;
    }


    public String getBabyWeight() {
        return babyWeight;
    }

    public void setBabyWeight(String babyWeight) {
        this.babyWeight = babyWeight;
    }

    public String getBabyLactation() {
        return babyLactation;
    }

    public void setBabyLactation(String babyLactation) {
        this.babyLactation = babyLactation;
    }

    public String getBabyRequireItem() {
        return babyRequireItem;
    }

    public void setBabyRequireItem(String babyRequireItem) {
        this.babyRequireItem = babyRequireItem;
    }

    public String getBabyEtc() {
        return babyEtc;
    }

    public void setBabyEtc(String babyEtc) {
        this.babyEtc = babyEtc;
    }

    public String getBoardConfirm() {
        return boardConfirm;
    }

    public void setBoardConfirm(String boardConfirm) {
        this.boardConfirm = boardConfirm;
    }

    public String getReplyCnt() {
        return replyCnt;
    }

    public void setReplyCnt(String replyCnt) {
        this.replyCnt = replyCnt;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public String getInsertDate() {
        return insertDate;
    }

    public void setInsertDate(String insertDate) {
        this.insertDate = insertDate;
    }
}

File: app/src/main/java/com/example/newbabyproject/Visit/ResultVisit.kt
package com.example.newbabyproject.Visit

import java.io.Serializable

class ResultVisit : Serializable {

    var seq : Int = 0
    var result: String? = null
    var userId: String? = null
    var userName: String? = null
    var userPhone: String? = null
    var babyName: String? = null
    var babyNum: String? = null
    var babyBirthDate: String? = null
    var babyBirthTime: String? = null
    var babyRelation: String? = null
    var parentId: String? = null
    var parentName: String? = null
    var visitNotice: String? = null
    var babyWeight: String? = null
    var babyLactation: String? = null
    var babyRequireItem: String? = null
    var babyEtc: String? = null
    var boardConfirm: String? = null
    var writeDate: String = ""
    var tempYn : String? = ""
    var reserveDate : String? = ""
    var replyCnt: String? = null
    var path1: String? = null
    var path2: String? = null
    var path3: String? = null
    var originalPath : String? = null
    var originalPath2 : String? = null
    var originalPath3 : String? = null
    var insertDate: String? = null

    constructor(
        userId: String?,
        userName: String?,
        userPhone: String?,
        babyName: String?,
        babyNum: String?,
        babyRelation: String?
    ) {
        this.userId = userId
        this.userName = userName
        this.userPhone = userPhone
        this.babyName = babyName
        this.babyNum = babyNum
        this.babyRelation = babyRelation
    }

    constructor(
        seq : Int,
        parentId: String?,
        parentName: String?,
        visitNotice: String?,
        babyWeight: String?,
        babyLactation: String?,
        babyRequireItem: String?,
        babyEtc: String?,
        writeDate: String,
        boardConfirm: String?,
        tempYn : String?,
        reserveDate : String?,
        path1: String?,
        path2: String?,
        path3: String?,
        originalPath : String?,
        originalPath2 : String?,
        originalPath3 : String?,
        //replyCnt: String?,
        insertDate: String?,
        babyName : String?
    ) {
        this.seq = seq
        this.parentId = parentId
        this.parentName = parentName
        this.visitNotice = visitNotice
        this.babyWeight = babyWeight
        this.babyLactation = babyLactation
        this.babyRequireItem = babyRequireItem
        this.babyEtc = babyEtc
        this.writeDate = writeDate
        this.boardConfirm = boardConfirm
        this.tempYn = tempYn
        this.reserveDate = reserveDate
        this.path1 = path1
        this.path2 = path2
        this.path3 = path3
        this.originalPath = originalPath
        this.originalPath2 = originalPath2
        this.originalPath3 = originalPath3
        //this.replyCnt = replyCnt;
        this.insertDate = insertDate
        this.babyName = babyName
    }

    override fun toString(): String {
        return "ResultVisit{" +
                "result='" + result + '\'' +
                ", seq = '" + seq + '\'' +
                ", userId='" + userId + '\'' +
                ", userName='" + userName + '\'' +
                ", userPhone='" + userPhone + '\'' +
                ", babyName='" + babyName + '\'' +
                ", babyNum='" + babyNum + '\'' +
                ", babyRelation='" + babyRelation + '\'' +
                ", parentId='" + parentId + '\'' +
                ", parentName='" + parentName + '\'' +
                ", visitNotice='" + visitNotice + '\'' +
                ", babyWeight='" + babyWeight + '\'' +
                ", babyLactation='" + babyLactation + '\'' +
                ", babyRequireItem='" + babyRequireItem + '\'' +
                ", babyEtc='" + babyEtc + '\'' +
                ", boardConfirm='" + boardConfirm + '\'' +
                ", writeDate='" + writeDate + '\'' +
                ", replyCnt='" + replyCnt + '\'' +
                ", path1='" + path1 + '\'' +
                ", path2='" + path2 + '\'' +
                ", path3='" + path3 + '\'' +
                ", insertDate='" + insertDate + '\'' +
                '}'
    }
}